omod/pom.xml

```diff
@@ -220,24 +220,7 @@
                 </executions>
             </plugin>
 
-            <plugin>
-                <groupId>de.saumya.mojo</groupId>
-                <artifactId>gem-maven-plugin</artifactId>
```

Comment:
Is this change needed for the ticket?

Reply:
Thanks @dkayiwa. Without making that change I could not run maven without errors.

Reply:
Can you revert those changes and i see the errors that you get?

Reply:
Thanks @dkayiwa . The errors I get are here https://snipboard.io/F83AoR.jpg with the full log here   https://pastebin.com/b2Mp9wrA

Reply:
Can you make a follow up commit with those changes reverted and we see?